cmake/tpl/zlib.cmake

```diff
@@ -1,5 +0,0 @@
-set(${lib_name}_libs libz.so)
-
-if(ENABLE_STATIC_TPL)
-  string(REPLACE ".so" ".a;" ${lib_name}_libs ${${lib_name}_libs})
-endif()
```

Comment:
I'm pretty sure ALE3D use `EANBLE_STATIC_TPL`, is that still the case @ptsuji ?

Reply:
Maybe @brbass is using this too?

Reply:
Yes, we're using ENABLE_STATIC_TPL.

Reply:
We are also using it.

Reply:
We don't need to explicitly bring zlib in as a target so we don't use this file anymore. The zlib tpl is included in the `CMAKE_PREFIX_PATH` instead.